kore/src/Kore/Repl.hs

```diff
@@ -174,6 +180,13 @@ runRepl tools simplifier predicateSimplifier axiomToIdSimplifier axioms' claims'
                 pure True
             else pure False
 
+    catchInterruptWithDefault :: a -> Simplifier a -> Simplifier a
+    catchInterruptWithDefault def sa =
```

Comment:
I think the lens-y (optical?) way of doing this is actually more complicated. Why not this?
```.haskell
catchInterruptWithDefault def =
    handle $ \UserInterrupt -> do
        liftIO $ putStrLn "Step evaluation interrupted."
        pure def
```